Locale/LocaleRu.py

```diff
@@ -0,0 +1,63 @@
+from dataclasses import dataclass
+from Locale.LocaleBase import Locale
+
+
+@dataclass(frozen=True)
+class LocaleRU(Locale):
+    greet = ['Здарова',
+             '! Добро пожаловать в InnoТягу!\n'
+             'Для того чтобы вывести доступные команды чиркани /help.\n'
+             'Для того чтобы вывести топ участников за всё время напишите /top.\n'
+             'Чтобы участвовать в топе, необходимо зарегистрироваться: /register.\n'
+             ]
+    schedule = 'Занятия иннотяги проходят в 19:00 по средам, пятницам и воскресеньям\n' \
+        'Среда - спина и бицепс, пятниа - ноги, воскресенье - трицепс и грудь\n' \
+        'Также иногда проводятся дополнительные занятия, ' \
+        'информацию по которым можно найти в основной беседе клуба: @innotyaga'
```

Comment:
```suggestion
    schedule = 'Да похуй всем Серёжа Допизделся) Лучше выложи расклад карт таро на твою ЛЕГЕНДАРНЕЙШЕЮ "Прошедшей ночью данные ребята ломились не в свою квартиру в 130 доме. При вопросе "чонада" свалили в закат. Наверное дом перепутали. Или город. Короче скоро новый год и иронию судьбы косплеели ребята...надеюсь...\nГоворят местные перепутали 130 и 114 дом." в Иннополисе мы все ждем'  \
    'Основная информация по вечно меняющемуся рассписанию в офф канале @innotyaga'
```
As per @kupautist's request...